<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Killer_Instinct</class>
 <widget class="QMainWindow" name="Killer_Instinct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Killer_Instinct</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagens.qrc">
    <normaloff>:/Imagens/Ícones/KI_Xbox_Logo_V3.png</normaloff>:/Imagens/Ícones/KI_Xbox_Logo_V3.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>150</width>
      <height>173</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Digite um PID: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="radioParar">
         <property name="text">
          <string>Parar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioContinuar">
         <property name="text">
          <string>Continuar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioMatar">
         <property name="text">
          <string>Matar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections/>
</ui>
